components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -100,12 +97,6 @@ const BillingSubscriptions = () => {
         dispatch(getCloudProducts(includeLegacyProducts));
         dispatch(getCloudCustomer());
 
-        if (!analytics) {
```

Comment:
analytics used to be used for the total_users and that logic was removed some time ago https://github.com/mattermost/mattermost-webapp/commit/144e5eef93cc8575fd5faf60e21b382aef3202ee#diff-a6189967fd50baa6b7db4f3163d5564d6ee31f5b2263f05e2b22d749b292d6e8L127